sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/processor.hpp

```diff
@@ -190,14 +190,18 @@ namespace Azure { namespace Messaging { namespace EventHubs {
     {
       Models::StartPosition startPosition = GetStartPosition(ownership, checkpoints);
 
+      // The consumers parameter is not stabilized across the lifetime of the partition client. Leak
+      // the partition for now.
       std::shared_ptr<ProcessorPartitionClient> processorPartitionClient
           = std::make_shared<ProcessorPartitionClient>(
               ownership.PartitionId,
               m_consumerClient->CreatePartitionClient(
                   ownership.PartitionId, {startPosition, m_processorOwnerLevel, m_prefetch}),
               m_checkpointStore,
               m_consumerClientDetails,
-              [&]() { consumers.erase(ownership.PartitionId); });
+              []() { /*
+                      consumers.erase(ownership.PartitionId);*/
```

Comment:
Do you intend to keep this commented out, or was this a thing that you were experimenting with during development?
If not, probably worth a comment, to answer the questions exactly as this one.

I think the comment about the leak above belongs here.

Reply:
This is intended to be commented out until I fix the processor. Right now the processor leaks resources like mad, this commented out code is there to remind me that stuff goes horribly wrong if I don't change it.

Since the processor isn't currently supported, the leaks aren't important, but the code in question is causing CI access violations.